frontend/src/Store/Actions/seriesActions.js

```diff
@@ -377,8 +379,56 @@ export const actionHandlers = handleThunks({
           });
         });
     }, MONITOR_TIMEOUT);
-  }
+  },
+
+  [UPDATE_SERIES_MONITOR]: function(getState, payload, dispatch) {
+    const {
+      id,
+      monitored,
+      monitor
+    } = payload;
+
+    const series = [];
+    const seriesToUpdate = { id };
+    if (payload.hasOwnProperty('monitored')) {
```

Comment:
New line before if block

Reply:
`if (monitored != null)` will pass if true or false, but I don't see a case where this will ever be set.

Reply:
Done